### ISSUE TITLE
Drop down menu in Feed using Android mobile devices

### ISSUE THREAD

**Issue by nivetheatha** [FightPandemics/FightPandemics#986]
Steps to create:
-Login with valid email id,Password.
-Go to feed.
-Select location drop down.
-It has to show results  related to location.(similarly if we choose provider or type.it has to show the respective results only)
-But it shows drop down menu again.
![Screenshot_20200703-154333_Samsung Internet](https://user-images.githubusercontent.com/41587211/86496632-4fab1e00-bd4c-11ea-80ec-486943466d57.jpg)

**Comment by idjevm** [comment]
Yeah this is something I have previously brought up, I mentioned that we should have a button called `Filters` on top, and then open this modal when that button is pressed, similar to what we have in Desktop Version. @ppagarawal Any thoughts?

**Comment by dabartlett** [comment]
Same on IOS (I tested using Iphone 6S).

**Comment by ppagarawal** [comment]
@idjevm this was supposed to be changed when @adisen would rework on the changes he merged. It is from the same 'pull request'. Reach out to me for more information.

**Comment by idjevm** [comment]
@ppagarawal so no actually, this is not in the scope of what Israel is doing, @adisen is just working to get the formatting back to how it was, which is this -> 
![image](https://user-images.githubusercontent.com/25408660/86506628-2b2c6180-bd9f-11ea-9935-603ac6ad1871.png)

but besides that, as @dabartlett mentioned, it makes no sense to have the filters as shown in this image, and then again ask the user to kinda select the filter they want. I think ideally we would want to have a button that say `Filters` like on the feed desktop view. and when the user clicks, then the menu is displayed. 
Something like this -> 
![image](https://user-images.githubusercontent.com/25408660/86506706-01276f00-bda0-11ea-979b-d16fe2211674.png)
But centered.

**Comment by idjevm** [comment]
Take a look at this gif -> it will make more sense @ppagarawal, a valid point that makes no sense to have these dropdowns, we should just have a `Filter by` button or text `Filter by` and then a button called `Filters` and that will open up the menu...

**Comment by idjevm** [comment]
https://share.getcloudapp.com/7KumzmGg

**Comment by ppagarawal** [comment]
@idjevm Please go ahead with the changes. It certainly makes no sense to have the user select the filters twice. I recommend that the implementation be as below but must open up the exact filter option when the modal window appears.

This is to let the user know what filters he can apply to.

![image](https://user-images.githubusercontent.com/6276256/86507649-8b4ef200-bdf7-11ea-8d83-8f6746e1ec18.png)

**Comment by idjevm** [comment]
This will require some code refactoring/logic thinking, If you look at the filters from the feed in the Desktop View, the user has the option to click on the "Filters" button and then the filters are displayed. Why not implement something like that?

**Comment by idjevm** [comment]
![image](https://user-images.githubusercontent.com/25408660/86508604-58810b80-bdaf-11ea-9170-0733e6ab0b96.png)
Like here, the user must click on filters first before the filters are shown

**Comment by idjevm** [comment]
+ as a user, what if I want to apply more filters, then I will have to click on a different filter, then view results, and then again and again and again. I think using the implementation like the feed desktop is much better, where the user clicks on the filters button and is able to see all the filters available, then the user can choose all the desired filters before applying, once the user is done choosing all the desired filters, the user can click on show results :) and this will apply all the filters selected in the filters menu. Much better implementation rather than having to apply one filter by one (can become annoying VERY fast)

**Comment by ppagarawal** [comment]
@idjevm I understand your point. But for now, lets stick to the design and I will work with the design to make any changes required in the future releases.
Also, selecting the filter criteria must directly open the criteria in the modal.

Please find the diagram below:
![image](https://user-images.githubusercontent.com/6276256/86510498-c3f9c600-be0d-11ea-8919-70301a24c7c7.png)

![image](https://user-images.githubusercontent.com/6276256/86510505-e986cf80-be0d-11ea-87b0-4cf2eefa0f92.png)

**Comment by idjevm** [comment]
Then based on this design, there doesn't need to be changes done, or do you want to just have pop up all the filters as it is right now but the section clicked extended?

**Comment by doheekim93** [comment]
@ppagarawal @idjevm Hey guys, what's the status of this ticket?

**Comment by ppagarawal** [comment]
This needs to be changed as discussed to provide better user experience. @idjevm can you or someone please take this up?

**Comment by ppagarawal** [comment]
> Then based on this design, there doesn't need to be changes done, or do you want to just have pop up all the filters as it is right now but the section clicked extended? 

The section click has to be extended
